File: java/Start/src/main/java/no/bouvet/solid/srpdip/OrderCancellationResponse.java
package no.bouvet.solid.srpdip;

public class OrderCancellationResponse extends ResponseMessage {
	private long orderId;
	public String orderState;
	public String message;

	public OrderCancellationResponse(long requestId, long orderId, String OrderState, String message) {
		super(requestId);
		this.orderId = orderId;
		this.orderState = orderState;
		this.message = message;
	}

	public long getOrderId() {
		return orderId;
	}

	public void setOrderId(long orderId) {
		this.orderId = orderId;
	}

	public String getOrderState() {
		return orderState;
	}

	public void setOrderState(String orderState) {
		this.orderState = orderState;
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}
}

File: cs/Refactor1/SRP_DI_Workshop/ServiceInterface/OrderCancellationResponse.cs
using System;

namespace SRP_DI_Workshop.ServiceInterface
{
    [Serializable]
    public class OrderCancellationResponse
    {
        public long RequestId { get; set; }
        public long OrderId { get; set; }
        public string OrderState { get; set; }
        public string Message { get; set; }
    }
}
